pyanaconda/display.py

```diff
@@ -346,20 +346,19 @@ def setup_display(anaconda, options):
             do_startup_x11_actions()
         except TimeoutError as e:
             log.warning("X startup failed: %s", e)
-            stdout_log.warning("X did not start in the expected time, falling back to text "
+            print("X did not start in the expected time, falling back to text "
```

Comment:
No idea why we used `stdout_log` in the first place. This might have some history and thus possibly reasons why this should not be changed?